cmd/libguestfs/entrypoint.sh

```diff
@@ -1,10 +1,3 @@
 #!/bin/bash -xe
 
-LIBGUESTFS_PATH=${LIBGUESTFS_PATH:=/tmp/guestfs}
-LIBGUEST_APPLIANCE=/usr/local/lib/guestfs/downloaded
-mkdir -p ${LIBGUESTFS_PATH}
-tar -Jxf ${LIBGUEST_APPLIANCE} -C ${LIBGUESTFS_PATH} --strip-components=1
-
-touch ${LIBGUESTFS_PATH}/done
-
```

Comment:
nit: is there any benefit in keeping this empty entrypoint.sh script instead of doing smth like `entrypoint = ["/bin/bash"]` in BUILD.bazel?

Reply:
@vasiliy-ul no, I could create a symlink. Initially, I was thinking of creating the done file, but then with @andreabolognani we decided to keep it simple. Not sure what is better between a symlink and an empty entrypoint, do you think the symlink is better?

Reply:
Hm... what symlink? I am probably a bit out of context here :)

I was just thinking that since the entrypoint.sh now simply calls bash, is there any reason why not making `/bin/bash` the actual entrypoint:
```
container_image(
    name = "libguestfs-tools-image",
    architecture = select({
        "@io_bazel_rules_go//go/platform:linux_arm64": "arm64",
        "//conditions:default": "amd64",
    }),
    base = ":version-container",
    directory = "/",
    entrypoint = ["/bin/bash"],
    visibility = ["//visibility:public"],
)
```
Then you could probably get rid of the script completely. Or am I missing smth?

Reply:
@vasiliy-ul yes, the reason for keeping the entrypoint.sh is if `virtctl guestfs` tries to use a different image version, it makes breaks. At least, downstream and upstream version of the `libguestfs-tools` image might not be aligned and a version still using the appliance in the old form that still requires the `entrypoint.sh`

Reply:
Sorry, but I still struggle a bit to understand the situation. This entrypoint.sh script is part of this exact image, right? If a different image is used, then it will be shipped with a different entrypoint. How can this entrypoint.sh affect the usage of a different image version?

Reply:
No the entrypoint is set here: https://github.com/kubevirt/kubevirt/blob/e19d67c0b8e6f310cf8c7a3153ac00e873c746a0/pkg/virtctl/guestfs/guestfs.go#L207 
and it is used to create the guestfs pod. I might be able to omit it and then it might pick the entrypoint defined in the image. I need to check

Reply:
Ah... So entrypoint.sh is explicitly passed as command to the manifest. I must have missed that. Now it becomes clear. Thanks for the explanation.

In general, I think it is still possible to drop the script. AFAIK, if you do not specify the command in the manifest, then the image's entrypoint will be run. But I do not insist on doing it within this PR. Just may be worth considering it for future optimizations.

Reply:
Ok, if you don't mind I'd like to do it in a separate PR